cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListTests.cy.js

```diff
@@ -250,7 +244,7 @@ describe("Payment Method list using Constraint Graph flow tests", () => {
 
       // payment method list which should only have credit with Stripe and Cybersource
       it("payment-method-list-call-test", () => {
-        let data = getConnectorDetails(globalState.get("connectorId"))[
+        let data = getConnectorDetails("stripe")[
```

Comment:
how do you make sure that it also runs for `cybersource`? i'm sure that this will run for `stripe` irrespective of connectors. 

the same applies applies else where

Reply:
This is only for getting the `Response` and `Request` `configs` which in `PaymentMethodList` will be same irrespective of any connector, as this is `connector agnostic`.
Moreover, we have implemented it for Stripe and Cybersource, So if the List runs fine for any 2 connectors it will run for any connectors, again as this is connector agnostic.